socorro/unittest/external/test_crashstorage_base.py

```diff
@@ -405,6 +421,45 @@ def test_poly_crash_storage_processed_crash_immutability(self):
             # is NOT affected.
             eq_(processed_crash['foo'], 'bar')
 
+    def test_polycrashstorage_processed_immutability_with_nonmutating(self):
+        """Verifies if a crash storage says it doesn't mutate the class that "
+        we don't do a deepcopy
+
+        """
+        n = Namespace()
+        n.add_option(
+            'storage',
+            default=PolyCrashStorage,
+        )
+        n.add_option(
+            'logger',
+            default=mock.Mock(),
+        )
+        value = {
+            'storage_classes': (
+                'socorro.unittest.external.test_crashstorage_base'
+                '.NonMutatingProcessedCrashCrashStorage'
```

Comment:
This test is identical to one of the existing ones except this one uses `NonMutatingProcessedCrashCrashStorage` and has a different assert at the end.